metadata-io/src/main/java/com/linkedin/metadata/entity/EntityService.java

```diff
@@ -1437,6 +1431,9 @@ public RollbackResult deleteAspect(String urn, String aspectName, Map<String, St
             survivingAspect == null ? ChangeType.DELETE : ChangeType.UPSERT, isKeyAspect, additionalRowsDeleted);
       } catch (URISyntaxException e) {
         throw new RuntimeException(String.format("Failed to emit the update for urn %s", urn));
+      } catch (IllegalStateException e) {
+        log.warn("Unable to find aspect, rollback result will not be sent. Error: {}", e.getMessage());
```

Comment:
same here. 
log.warn("Unable to find aspect, rollback result will not be sent. ", e) should work

Reply:
Should we be spamming full stacktraces for warns?

Reply:
sg. let's keep it as is